src/App.jsx

```diff
@@ -3,23 +3,16 @@ import theme from "./themes/theme";
 import { BrowserRouter } from "react-router-dom";
 import Footer from "./components/footer";
 import Home from "./pages/home";
-import HeroSection from "./components/hero-section";
-import Header from "./components/header"
-import Banner from "./pages/home/banner-section"
-import NewArrivalsSection from "./pages/home/new-arrivals-section";
-import BrandsSection from "./pages/home/brands-section";
+import Header from "./components/header";
 
 
 function App() {
   return (
     <BrowserRouter>
       <ThemeProvider theme={theme}>
-        {/* <Header /> */}
-        <HeroSection/>
-        <NewArrivalsSection/>
-        <BrandsSection/>
-        <Banner/>
-        <Footer/>
+        <Header />
+        <Home />
```

Comment:
you removed these components from App, but I can't find them added in Home component.

 ```
<HeroSection/>
        <NewArrivalsSection/>
        <BrandsSection/>
        <Banner/>
```

Reply:
Zaid will put the home page together and create the routing, I only put the components that I created